src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -3311,6 +3311,22 @@ namespace winrt::TerminalApp::implementation
 
             const IInspectable unused{ nullptr };
             _SetAsDefaultDismissHandler(unused, unused);
+
+            // TEMPORARY SOLUTION
+            // If the connection has requested for the window to be maximized,
+            // manually maximize it here. Ideally, we should be _initializing_
+            // the session maximized, instead of manually maximizing it after initialization.
+            // However, because of the current way our defterm handoff works,
+            // we are unable to get the connection info before the terminal session
+            // has already started.
+
+            // Make sure that there were no other tabs already existing (in
+            // the case that we are in glomming mode), because we don't want
```

Comment:
Crazy idea: if we're in glomming mode and a defterm connection requests to be maximized, should we not glom that connection? Open it in a new window instead and make _that_ window maximized?

I guess then we're opening up the possibility for needing to check if a window is maximized and (if that's the case) glomming to that one if maximized was requested.

Reply:
Unfortunately, that decision cannot be made at this point. We've already received an incoming handoff and established its pipe hookup to a `ConptyConnection`; we aren't equipped to yeet that session anywhere else without a lot of wailing and gnashing of teeth.

Reply:
(We have to receive the incoming handoff to determine whether to glom it, but we need to start the Windows Terminal Handoff COM server (and report back to COM) to _get_ that information . . . and by the time we've done that, the Monarch has selected a Peasant to handle the connection already.)